srcpkgs/gmsh/template

```diff
@@ -16,26 +16,27 @@ license="GPL-2.0-or-later"
 homepage="https://gmsh.info"
 changelog="http://gmsh.info/CHANGELOG.txt"
 distfiles="https://gmsh.info/src/gmsh-${version}-source.tgz"
-checksum=0f2c55e50fb6c478ebc8977f6341c223754cbf3493b7b0d683b4395ae9f2ad1c
+checksum=e27f32f92b374ba2a746a9d9c496401c13f66ac6e3e70753e16fa4012d14320e
+
+build_options="fltk hxt numpy zipper"
```

Comment:
I don't believe the `numpy` build option is functional. Looking at `CMakeLists.txt`, setting `-DENABLE_NUMPY=ON` is only functional with both `-DENABLE_PRIVATE_API=ON` and `-DENABLE_WRAP_PYTHON=ON`. In addition, this requires `swig`, `python3-numpy` and probably `python3-devel`. You will also probably have to set `NUMPY_INC` to make this work properly for cross builds.

If you don't want to sort out the details, I suggest you just drop the `numpy` build option because it isn't on by default anyway.